src/js/modules/userInfo/reducer.js

```diff
@@ -0,0 +1,39 @@
+import * as actionTypes from './actiontypes';
+import _ from 'lodash';
+
+// const initialState = {
+//     accessToken: '',
+//     userData: {},
+// };
+
+const initialState = {
+    accessToken: 'eyJhbGciOiJFUzI1NiIsIng1dCI6IkQ0QUU4MjQ2RDYyNTBFMTY5Njg4NDFCREY4Nzc2MTI4NUMwNUJCMUYifQ.eyJvYWEiOiI3Nzc3NyIsImlzcyI6Im9hIiwiYWlkIjoiMTEwIiwidWlkIjoiSXlERjVicGJkRTRhTEtObGUxU2RCZz09IiwiY2lkIjoiSXlERjVicGJkRTRhTEtObGUxU2RCZz09IiwiaXNhIjoiVHJ1ZSIsInRpZCI6IjIwMDIiLCJzaWQiOiJlYWI0ZmNkOGY4YTE0OTAyYTQ5NDg5NDQ4NzY4MTYwOCIsImRnaSI6IjgyIiwiZXhwIjoiMTUxMTMzNTUwNyJ9.aiS3tQn-GmdMGbI035j4kc6KtvHKGjsWMQctB_cDOfOie0Xw-L_Dxi5hVCdxZw0AzgyFKiOCH75-zrT84GMQBA',
```

Comment:
access token shouldn't be hardcoded here

Reply:
access token shouldn't be uploaded to any public repository at all.